chevah/brink
# Add support for direct URL download page.

Scope
=====

This updates the code for generating the download page to allow creating external URL download links.


Changes
=======

Update the URL generation to be moved outside of the template.
The URL is not generated when the configuration already has a precomputed URL.


How to try and test the changes
===============================

reviewers: @dumol 

This PR is FYI.

You can see the result in the chavah/sftpplus.com PR ...and soon in https://github.com/chevah/server/pull/1722
